src/databricks/labs/pytester/fixtures/catalog.py

```diff
@@ -297,8 +297,8 @@ def test_catalog_fixture(make_catalog, make_schema, make_table):
     ```
     """
 
-    def create() -> CatalogInfo:
-        name = f"dummy_C{make_random(4)}".lower()
+    def create(name: str = "") -> CatalogInfo:
```

Comment:
The convention for `create()` methods is:

 - Keyword-only arguments.
 - A default of `None` if they're optional.
 - `catalog_name` instead of `name` (for better composability).

The argument also needs to be described in the docstring for `make_catalog` above to ensure that it ends up in the project documentation.

Reply:
`name` -> `catalog_name` is not similar to `make_schema`